cmd/bootkube/render.go

```diff
@@ -195,3 +224,28 @@ func altNamesFromURLs(urls []*url.URL) *tlsutil.AltNames {
 	}
 	return &an
 }
+
+func pickDNSServiceIP(serviceCIDR string) (string, error) {
+	ip, ipnet, err := net.ParseCIDR(serviceCIDR)
+	if err != nil {
+		return "", err
+	}
+	// pick the 10th IP as a conventional default
+	for i := 0; i < 10; i++ {
+		incIPv4(ip)
```

Comment:
I'm happy to do it however. As long as it consistently gives the same value it should be ok right? The particular IP chosen was an arbitrary choice